src/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/dcdd89c74c.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <div class="main">
+      <header>
+        <div class="top-nav">
+          <div class="top-part">
+            <h2>A Web Page</h2>
+          </div>
+          <div class="bottom-part">
+            <i class="fa-solid fa-left-long fa-3x white"></i>
+            <i class="fa-solid fa-right-long white fa-3x"></i>
+            <i class="fa-solid fa-xmark fa-3x"></i>
+            <i class="fa-solid fa-house fa-2x white"></i>
+            <input type="text" placeholder="https://" class="http-input" />
+            <div class="search-bar">
+              <input
+                type="text"
+                placeholder="Search..."
+                id="search-text-input"
+                class="curved-input"
+              /><i class="fa-solid fa-magnifying-glass"></i>
+            </div>
+          </div>
+        </div>
+      </header>
+      <h2 class="leader">Leaderboard</h2>
+      <div class="container">
```

Comment:
- [ ] Example case for using flex as a class: `<div class="container flex">`
Create a flex class now in your CSS.
Check every element in your html that you want it to be display flex and pass flex class to them